add more package info

ye-zhi opened the pull request:
add more package information based on Ross's comment

ye-zhi committed:
add more package info based on Ross's comment

ye-zhi committed:
fix spelling

timothy1191xa commented:
@ye-zhi 
In the fix spelling commit did you delete the two lines (-babel==2.0.1
-statsmodels==0.6.1) ?

ye-zhi commented:
@timothy1191xa  Yes. I somehow did not type in the first two characters of "nibabel" so I changed it. For the "statsmodels", I added it first, but I was not sure if our project will need it, so I just deleted it.

timothy1191xa commented:
ok! I'm just going to merge it then
